vos/vos/vos.py

```diff
@@ -1421,13 +1421,7 @@ def __init__(self, vospace_certfile=None, root_node=None, conn=None,
                               vospace_token=vospace_token,
                               http_debug=http_debug)
 
-        self.protocol = vos_config.get('transfer', 'protocol')
-        if self.protocol not in self.VO_TRANSFER_PROTOCOLS:
-            raise SystemError(
-                'Unsupported protocol {}. Valid protocols: {}. Update {}'.
-                format(self.protocol, self.VO_TRANSFER_PROTOCOLS,
-                       _CONFIG_PATH))
-
+        self.protocols = Client.VO_TRANSFER_PROTOCOLS
```

Comment:
Without doing anything the `protocol` keyword will remain in the config file falsely suggesting that the code uses it. It should check and if it exists it should probably remove it - with or without user's consent. User consent is a policy decision - asked Séverin whether we should do it or not. Another alternative is to comment it out in the config file and add text saying that this is no longer supported and the user should remove it from the config file.

Reply:
I think your 2nd suggestion is better. I'll update the default-vos-config file.

Then there is no way for a user to specify unsecure protocol (e.g. HTTP) unless we add an option to the command line. Should we consider that?